src/index.js

```diff
@@ -154,7 +153,21 @@ export default class Headroom extends Component {
   }
 
   render () {
-    let style = {
+    const divProps = Object.assign(this.props)
+    delete divProps.onUnpin
+    delete divProps.onPin
+    delete divProps.onUnfix
+    delete divProps.disableInlineStyles
+    delete divProps.disable
+    delete divProps.parent
+    delete divProps.children
+    delete divProps.upTolerance
+    delete divProps.downTolerance
+    delete divProps.pinStart
+
+    const { style, wrapperStyle, ...rest } = divProps
```

Comment:
I like this. I'll merge this PR and would you like to make a PR for your change after that? That's mostly if you want the Git attribution. If you don't care much I'll just copy in your change myself.